test/bats/tmp/.gitignore

```diff
@@ -0,0 +1 @@
+#ignore
```

Comment:
@josephharrington tmp, fixtures, etc. directories are part of the BATS framework. I will add comment in the placeholder gitignore files to clarify that.